src/components/DpUploadFiles/DpUpload.vue

```diff
@@ -255,7 +259,7 @@ export default {
     })
 
     this.uppy.on('restriction-failed', () => {
-      dplan.notify.warning(Translator.trans(this.allowedFileTypesWarning))
+      dplan.notify.warning(this.allowedFileTypesWarning)
```

Comment:
Is it safe to remove this?

Reply:
Yes, we use in this prop `allowedFileTypesWarning` Trans key `default: de().strings.warningFileType`

Reply:
So at the moment, we are never passing in a value for this prop?

Reply:
no